inscriptions/ISic001192.xml

```diff
@@ -190,22 +195,28 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140676">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#MH">
                 <ab>
-                    <lb n="1"/>
-
-
-                    οἱ ἀλειφόμε
-                    <supplied reason="lost">νοι</supplied>
-                    <lb n="2"/>Ἀ<supplied reason="omitted">γ</supplied>άθαρχον<lb n="3"/> Ἡρακλείου.
-                    <lb n="4"/>
+                    <lb n="1"/>οἱ ἀλειφόμε<supplied reason="lost">νοι</supplied>
+                    <lb n="2"/><name>Ἀ<supplied reason="lost">γ</supplied>άθαρχον</name>
+                    <lb n="3"/><name>Ἡρακλείου</name>.
                 </ab>
             </div>
             <div type="apparatus">
-                <p><!--external apparatus criticus (if applicable)--></p>
+                <listApp>
+                    <app>
+                        <note>Text according to Kaibel in IG</note>
+                    </app>
+                    <app loc="1">
+                        <note>Gualt: ΟΙΛΕΙΦΟΜΕ</note>                      
```

Comment:
I think Gualtherus has ΟΙΑΛΕΙΦΟΜΕ, which is actually therefore the same as IG's text and doesn't need to be reported?